Formula/ansi.rb

```diff
@@ -0,0 +1,17 @@
+class Ansi < Formula
+  desc "This bash script is designed to help you colorize words and bits of text"
+  homepage "https://github.com/fidian/ansi"
+  url "https://github.com/fidian/ansi.git",
+    :using => :git,
+    :revision => "3624905213bd89f1023c1190df25069acc14abd5"
```

Comment:
We require a tagged release, not just a commit. Also, tarballs are preferred over git checkouts unless they are needed for a reason, e.g. submodules.

As the most recent tag is almost two years old I suggest that you ask upstream to tag a new release.